Homework/Duy_Truong/217_contain_duplicate.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @param {number[]} nums
+ * @return {boolean}
+ */
+var containsDuplicate = function(nums) {
+    let arr = Array.from(new Set(nums))
```

Comment:
this looks neat but can you try another way that not using built-in function?

Reply:
I've fixed